### PR TITLE
Border-bottom fix and fixed text in Features section

### PR DESCRIPTION
**Pull request by sboldyreva**
Fixes:
1. Features Section: text "Effortless to switch to" wasn't displayed properly
2. Commercial Support border-bottom color differed from other sections
image

### CODE REVIEW

**Comment by bennyvasquez** [comment]
@sboldyreva it looks like this one didn't actually fix that problem, but it might be related to the conflict I tried to resolve. https://d6d9978b.almalinux-org.pages.dev/ is the deployed testing. Mind taking another look when you get a second?

**Comment by sboldyreva** [comment]
@bennyvasquez hmmm interesting, just checked and it fixes the problem locally (as I showed on a screenshot)

**Comment by bennyvasquez** [comment]
@jonathanspw @codyro it looks like this might be related to the automated running of find_missing_i18n_strings.py. When pull the code in this PR it displays correctly, but when it deploys via cloudflare it fails to show the text under 'Effortless to switch to'. I ran the script on my laptop, and got this output:

`elita:sboldyreva_almalinux.org benny$ python3 find_missing_i18n_strings.py 
TRANSLATION ERROR: Switching distributions is usually costly and time-consuming, but that’s not the case when switching from CentOS and other RHEL® derivatives to AlmaLinux OS."}} {{ i18n "Switching is easy
Adding 'Switching distributions is usually costly and time-consuming, but that’s not the case when switching from CentOS and other RHEL® derivatives to AlmaLinux OS."}} {{ i18n "Switching is easy'
Removing 'Switching distributions is usually costly and time-consuming, but that’s not the case when switching from CentOS and other RHEL® derivatives to AlmaLinux OS.'
Removing 'Switching is easy'
elita:sboldyreva_almalinux.org benny$`

It seems like that's showing it adding in lines without a comma, but I'm not positive. Are there deeper logs you can look at on the cloudflare side?

**Comment by codyro** [comment]
Just commenting in case the above was missed :)

**Review by codyro — COMMENTED** [review]
It helps if I click the `Submit review` button 😓

**Comment by sboldyreva** [comment]
@codyro please check if I did it right and it works

**Comment by codyro** [comment]
Looks good to me! If you can merge in/sync your branch from `master` to pull in the `en.json` changes, I think this will be safe to merge this in :).

**Comment by bennyvasquez** [comment]
@codyro It's still not showing the text under 'Effortless to switch to', but it's in both the language files and the index page and displays correctly when I build the branch locally. https://f0e38eed.almalinux-org.pages.dev/ what am I missing?

File: layouts/index.html [hunk 1]
```diff
@@ -474,7 +474,7 @@ <h4 class="fw-bold mb-0 pb-2">{{ i18n "Fully supported, always free" }}</h4>
                         <div>
                             <h4 class="fw-bold mb-0 pb-2">{{ i18n "Effortless to switch to" }}</h4>
                             <p>
-                                {{ i18n "Switching distributions is usually costly and time-consuming, but that’s not the case when switching from CentOS and other RHEL® derivatives to AlmaLinux OS."}} {{ i18n "Switching is easy" }} - <a href="https://github.com/AlmaLinux/almalinux-deploy">{{ i18n "read how" }}.</a>
+                                {{ i18n "Switching distributions is usually costly and time-consuming, but that's not the case when switching from CentOS and other RHEL® derivatives to AlmaLinux OS."}} {{ i18n "Switching is easy" }} - <a href="https://github.com/AlmaLinux/almalinux-deploy">{{ i18n "read how" }}.</a>
```
Comment (bennyvasquez) [inline layouts/index.html:477]:
```suggestion
                                {{ i18n "Switching distributions is usually costly and time-consuming, but that's not the case when switching from CentOS and other RHEL® derivatives to AlmaLinux OS." }} {{ i18n "Switching is easy" }} - <a href="https://github.com/AlmaLinux/almalinux-deploy">{{ i18n "read how" }}.</a>
```

Comment (bennyvasquez) [reply layouts/index.html:477]: Adding this missing space makes find_missing_i18n_strings.py happy.

**Review by bennyvasquez — APPROVED** [review]
The adding of the space fixed the deploy problems. Approving! https://7a3fca66.almalinux-org.pages.dev/

**Pull request merged**